common/src/main/java/io/netty/util/DomainMappingBuilder.java

```diff
@@ -80,7 +79,7 @@ public DomainMappingBuilder(int initialCapacity, V defaultValue) {
      * @return new {@link DomainNameMapping} instance
      */
     public DomainNameMapping<V> build() {
-        return new ImmutableDomainNameMapping<V>(this.defaultValue, this.map);
+        return new ImmutableDomainNameMapping<V>(defaultValue, map);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Avoid instantiation through private constructors from outside of the constructors class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AAccessorClassGeneration)